webapp/components/compat-2021.js

```diff
@@ -32,8 +32,18 @@ class Compat2021 extends PolymerElement {
           experimental features enabled.
         </template>
       </p>
-      <compat-2021-feature-chart stable="[[stable]]"></compat-2021-feature-chart>
-      <p>TODO: Test results table</p>
+
+      <!-- TODO: Use a paper-toggle instead -->
+      <div>
+        <label for="stableVersionCheckbox">Show results from Stable browser releases:</label>
+        <input id="stableVersionCheckbox" type="checkbox" />
```

Comment:
Can we use the toggle component, like Robert suggested?

Reply:
Are you thinking of a toggle that looks like:

```
stable (   (*)) experimental
[click]
stable ((*)   ) experimental
```

I.e. has text on either side? Or are you thinking of something different?

One different approach we could also take is the experimental/stable buttons you find on the BSF graph on https://wpt.fyi, where they're separate buttons but they can't both be selected.

Reply:
I guess a toggle indeed isn't great, it's better if one can understand the current state, not just that it's *not* stable, if that were the name of the toggle.

The buttons on the BSF graphs seem OK. They take up a lot of horizontal space, but maybe that's OK/